mdsuite/calculators/green_kubo_ionic_conductivity.py

```diff
@@ -293,21 +283,16 @@ def plot_data(self, data):
             self.plot_array.append(fig)
 
     def run_calculator(self):
-        """
-        Run analysis.
-
-        Returns
-        -------
-
-        """
+        """Run analysis."""
         self.check_input()
         # Compute the pre-factor early.
         self._calculate_prefactor()
 
         dict_ref = str.encode(
             "/".join([self.loaded_property.name, self.loaded_property.name])
         )
-
+        self.count = 0
```

Comment:
Nothing that needs to be changed, but making a run variable a class attribute somehow doesn't feel right.

Reply:
It is updated in methods throughout the class